### ISSUE TITLE
parseToAliases will not parse two appended emojis

### ISSUE THREAD

**Issue by erlichsefi** [vdurmont/emoji-java#31]
it i have to emojiis one by one of instance: 
`����`

the output is 

`:grinning:�� `

while the result should be : 
`:grinning::joy`:

**Comment by krishnakumarcitrus** [comment]
I too have a similar issue when trying to parse two emoji's placed without a space in between.(Reverse case of this)
I tried to convert  :grinning::joy: to Unicode.

**Comment by freva** [comment]
The original problem is the same as #30, which is fixed in #29 if it gets accepted.

I'm not sure what the problem with the reverse case is?
```java
String alias = ":grinning::joy:";
String unicode = "\uD83D\uDE00\uD83D\uDE02"; //:grinning::joy: as Unicode
System.out.println(EmojiParser.parseToUnicode(alias).equals(unicode));
```
gives `true`.

**Comment by vdurmont** [comment]
I merged #29 (thanks again @freva ) so this should be fixed in v3.1.1.
I just released it and it should be available on maven central within a few hours.

Please reopen if you still have this issue after upgrading.
